pkg/log/log.go

```diff
@@ -85,19 +85,38 @@ func InitLoggers(globalCfg GlobalConfig, subCfgs map[string]GlobalConfig, opts .
 		if globalCfg.EcsIntegration {
 			cfg.Zap.EncoderConfig = ecszap.ECSCompatibleEncoderConfig(cfg.Zap.EncoderConfig)
 		}
-		logger, err := cfg.Zap.Build(opts...)
-		if err != nil {
-			return err
-		}
+
+		var cores []zapcore.Core
 		if cfg.StderrRedirectFile != nil {
 			stderrF, err := os.OpenFile(*cfg.StderrRedirectFile, os.O_WRONLY|os.O_CREATE|os.O_SYNC|os.O_APPEND, 0600)
 			if err != nil {
 				return err
 			}
-			if err := redirectStderr(stderrF); err != nil {
-				return err
-			}
+
+			cores = append(cores, zapcore.NewCore(
+				zapcore.NewJSONEncoder(cfg.Zap.EncoderConfig),
+				zapcore.AddSync(stderrF),
+				cfg.Zap.Level))
 		}
+
+		consoleCfg := zap.NewDevelopmentConfig()
+		consoleCfg.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
+		cores = append(cores, zapcore.NewCore(
+			zapcore.NewConsoleEncoder(consoleCfg.EncoderConfig),
+			zapcore.Lock(os.Stderr),
+			zap.InfoLevel))
+
```

Comment:
so looks like add a new console encoder to process the output in human-friendly format?

i assume you already run locally to verify the log format in the file has remained the same?

Reply:
yes,  add console encoder.  I run in test-net, log format is the same with older